<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Processes Schedular</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{
background: #02475e;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QTextBrowser" name="textBrowser">
        <property name="font">
         <font>
          <family>Century Gothic</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border-radius: 5px;
	border: 2px;
	padding: 1px;
}</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
color:white;
}</string>
          </property>
          <property name="text">
           <string>Process Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="proccesName">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border-radius: 5px;
	border: 2px;
	padding: 1px;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
color:white;
}</string>
          </property>
          <property name="text">
           <string>Arrival Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="arrivalTime">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border-radius: 5px;
	border: 2px;
	padding: 1px;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
color:white;
}</string>
          </property>
          <property name="text">
           <string>Burst Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="burstTime">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border-radius: 5px;
	border: 2px;
	padding: 1px;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Century Gothic</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
margin-top : 10px;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Current Proccesses in queue</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <family>Century Gothic</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: green;
color:white;
border-radius: 4px;
height: 30px;
}</string>
        </property>
        <property name="text">
         <string>Add new process</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="font">
         <font>
          <family>Century Gothic</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: #687980;
color:white;
border-radius: 4px;
height: 30px;
}</string>
        </property>
        <property name="text">
         <string>Simulate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
